easybuild/easyconfigs/p/Python/Python-2.7.12-intel-2016b.eb

```diff
@@ -0,0 +1,132 @@
+name = 'Python'
+version = '2.7.12'
+
+homepage = 'http://python.org/'
+description = """Python is a programming language that lets you work more quickly and integrate your systems
+ more effectively."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+
+# python needs bzip2 to build the bz2 package
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('zlib', '1.2.8'),
+    ('libreadline', '6.3'),
+    ('ncurses', '6.0'),
+    ('SQLite', '3.13.0'),
+    ('Tk', '8.6.5', '-no-X11'),
```

Comment:
this hasn't been an issue up until now, why should it suddenly become a problem?

anyway, we need to work towards a single Tk that does not rely on system X11 deps anyway, so we should probably just drop the versionsuffix here, and then later enhance the Tk easyconfig to include an X11 bundle (once we have that in place)...